L/libnl/build_tarballs.jl

```diff
@@ -13,10 +13,12 @@ sources = [
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/libnl-*/
+cd $WORKSPACE/srcdir
+cd libnl-*
 ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
 make -j${nproc}
 make install
+install_license COPYING
```

Comment:
Seems like this may still be relevant?

Reply:
No, the license is already automatically installed